lib/arjdbc/vertica/adapter.rb

```diff
@@ -258,7 +269,7 @@ def vertica_column_types_for(table_name, column_names)
     end
 
     def vertica_column_type_for(table_name, column_name)
-      column = vertica_memoized_columns(table_name).find { |col| col.name == "#{column_name}" }
+      column = vertica_memoized_columns(table_name).find { |column| column.name == "#{column_name}" }
```

Comment:
Nitpick: Let's avoid the shadow naming here. That's probably why it was `col` before (to avoid the outer `column = `).

Reply:
ahh i was wondering about that